packages/flutter_tools/test/commands.shard/hermetic/attach_test.dart

```diff
@@ -83,6 +84,7 @@ void main() {
     group('with one device and no specified target file', () {
       const int devicePort = 499;
       const int hostPort = 42;
+      final int year3000 = DateTime(3000).millisecondsSinceEpoch;
```

Comment:
Alternatively you could make it relative to now to avoid bugs where the underlying library tries to do something cute with this `validUntil` and is expecting something reasonable.
```suggestion
      final int future = DateTime.now().add(const Duration(days: 1)).millisecondsSinceEpoch;
```